### PR TITLE
Update calibration description to make flatness clear

### PR DESCRIPTION
**Pull request by Luxonis-Brandon**

### CODE REVIEW

File: source/pages/calibration.rst [hunk 1]
```diff
@@ -13,7 +13,9 @@ For the :ref:`BW1097 - RaspberryPi Compute Module` and :ref:`BW1098OBC - USB3 wi
 pre-calibrated - but you may want to re-calibrate for better quality in your installation (e.g. after mounting the board to something),
 or if the calibration quality has started to fade over use/handling.
 
-Below is a quick video showing the (re-) calibration of the :ref:`BW1097 - RaspberryPi Compute Module`.
+Below is a quick video showing the (re-) calibration of the :ref:`BW1097 - RaspberryPi Compute Module`.  In short, the calibration uses the intersections to determine the orientation and distance of the checkerboard.  So the greatest accuracy will be obtained by a clear print or display of the provided checkerboard image on a flat plane. 
+
+The flatness of the calibration checkerboard is very important.  Do not use curved monitors, or calibration targest that have any 'waviness'.  So if you print the checkerboard, please make sure affix the sheet to a known flat surface, without any waves.  That said, using a laptop with a flat monitor is usually the easiest technique.
```
Comment (WhitherGo) [inline source/pages/calibration.rst:18]: Spelling: "Do not use curved monitors, or calibration targest" should be "targets."
Grammar: "please make sure affix the sheet" should be "please make sure to affix the sheet."

File: source/pages/calibration.rst [hunk 2]
```diff
@@ -13,7 +13,9 @@ For the :ref:`BW1097 - RaspberryPi Compute Module` and :ref:`BW1098OBC - USB3 wi
...
 
 Watching the video below will give you the steps needed to calibrate your own DepthAI.  And for more information/details on calibration options,
 please see the steps below and also :code:`./calibrate.py --help` which will print out all of the calibration options.
```
Comment (WhitherGo) [inline source/pages/calibration.rst:21]: Grammar: "calibrate your own DepthAI.  And for more information" should drop the "And" to begin the second sentence "For more information."

File: source/pages/calibration.rst (hunk 1 above)
Comment (Luxonis-Brandon) [reply source/pages/calibration.rst]: Fixed - thanks!

File: source/pages/calibration.rst (hunk 2 above)
Comment (Luxonis-Brandon) [reply source/pages/calibration.rst:21]: Fixed - thanks!

**Pull request merged**